example/src/App.tsx

```diff
@@ -73,10 +71,8 @@ const App = () => {
   ) : (
     <>
       <Editor
-        errorHandler={(message: string) => {
-          setHasError(true);
-          setErrorMessage(message.toString());
-        }}
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        errorHandler={() => {}}
```

Comment:
I checked soem other places are still using this `editor.errorHandler`, do we need to remove it completely? and all the corresponding places of usage?

Reply:
Why don't we move this old errorHandler since it seems doing nothing here?

Reply:
I attempted to remove the editor.errorHandler, but it led to numerous crashes throughout the application because of its extensive usage in various scenarios.  I believe it's not ideal to remove it at this stage. I suggest keeping it for now.